session/session.go

```diff
@@ -1271,6 +1272,11 @@ func CreateSession(store kv.Storage) (Session, error) {
 	}
 	privilege.BindPrivilegeManager(s, pm)
 
+	bm := &bindinfo.BindManager{
+		GlobalAccessor: do.BindHandle(),
```

Comment:
Why do we need this interface? Why not obtain the handle in executor?

Reply:
For better abstraction

Reply:
I don't think so. The interface here just brings extra burden to understand the code.

Reply:
BindManager will have query feature, sessionBindHandle in the next pr. So we need a interface in session, then we only need to access the manager rather than access a global bind cache and a session bind cache.

Reply:
Then please add it when necessary.